plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/LibrarySelectorGroup.java

```diff
@@ -58,30 +58,33 @@
   private final ListenerList/* <ISelectedChangeListener> */ listeners = new ListenerList/* <> */();
 
   public LibrarySelectorGroup(Composite parentContainer, String groupName) {
-    this(parentContainer, groupName, true);
+    this(parentContainer, groupName, Messages.getString("appengine.libraries.group"), true); //$NON-NLS-1$
```

Comment:
Would you rename the key to remove `appengine`?

Reply:
I'll push this up to the callers.